Cargo.toml

```diff
@@ -59,3 +58,6 @@ u64_backend = ["curve25519-dalek/u64_backend"]
 u32_backend = ["curve25519-dalek/u32_backend"]
 fiat_u64_backend = ["curve25519-dalek/fiat_u64_backend"]
 fiat_u32_backend = ["curve25519-dalek/fiat_u32_backend"]
+
+[patch.crates-io]
+curve25519-dalek = { git = "https://github.com/mobilecoinfoundation/curve25519-dalek.git", rev = "8791722e0273762552c9a056eaccb7df6baf44d7" }
```

Comment:
This can be removed once https://github.com/dalek-cryptography/curve25519-dalek/pull/383 is merged and published.